py3status/module.py

```diff
@@ -350,6 +351,29 @@ def set_module_options(self, module):
 
             self.i3s_module_options['separator_block_width'] = sep_block_width
 
+        for name in ('background', 'border'):
+            if name in mod_config:
+                param = mod_config[name]
+                if not isinstance(param, basestring):
+                    err = 'invalid "{}" attribute, '.format(name)
+                    err += 'should be a string'
+                    raise TypeError(err)
+                self.py3_module_options[name] = Py3(self)._get_color(param)
+                self.block_options.append(name)
```

Comment:
@tobes Look at this part. I don't know if `Py3(self)._get_color(param)` is ideal.

Reply:
yes this is very bad

a) Instantiating `Py3` just to access a function is a performance overhead

b) class methods starting with an underscore `_` are private and should not be accessed outside of the class

I'll make a PR to make the function globally available

Reply:
I'll wait for this. :-)